docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_managed_identity/linux/_index.md

```diff
@@ -0,0 +1,405 @@
+
+The following README will guide you on how to work with the Hybrid Instance Metadata Service on an Azure Arc-enabled server, and authenticate the system assigned Managed Identity against Azure APIs.
+
+In this guide you will use Bash commands and REST API calls to:
+
+- Query the Hybrid Instance Metadata Service (HIMDS)
+- Get challenge tokens and resource tokens
+- Use the main Azure Resource Manager REST API
+- Read a secret from an Azure Key Vault
+- Upload a blob to an Azure Storage Account
+
+## Prerequisites
+
+* An onboarded Ubuntu VM or server running on-premises or other clouds connected to Azure Arc
+
+    If you haven't, this repository offers you a way to do so in an automated fashion:
+
+    * **[GCP Ubuntu instance](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/gcp/gcp_terraform_ubuntu/)**
+    * **[AWS Ubuntu EC2 instance](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/aws/aws_terraform_ubuntu/)**
+    * **[Azure Ubuntu VM](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/azure/azure_arm_template_linux/)**
+    * **[VMware vSphere Ubuntu VM](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/vmware/vmware_terraform_ubuntu/)**
+    * **[Vagrant Ubuntu box](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/vagrant/local_vagrant_ubuntu/)**
+
+* Tagging
+
+    Add a tag to your Azure Arc-enabled server.
+
+    | Tag | Value |
+    |---|---|
+    | scenario | Managed Identity |
+
+    The tag will be seen later in the guide when accessing the Hybrid Instance Metadata Service.
+
+    ![Screenshot showing the tagged Azure Arc-enabled server](./01.png)
+
+* jq and curl
+
+    SSH on to the Ubuntu server and install the jq and curl packages.
+
+    ```shell
+    sudo apt-get update && sudo apt-get install -y jq curl
+    ```
+
+## Azure Key Vault and Azure Storage account
+
+The scenario will demonstrate how to integrate with Azure services by accessing a secret in an Azure Key Vault and uploading a blob to an Azure Storage account.
+
+Login to your Azure account to run the commands. (This can also be done in [Azure Cloud Shell](https://shell.azure.com/bash).)
+
+* Set variables
+
+  ```shell
+  rg=arc_managed_identity_scenario
+  loc=westeurope
+  ```
+
+* Create the resource group
+
+  ```shell
+  az group create --name $rg --location $loc
+  rgid=$(az group show --name $rg --query id --output tsv)
+  ```
+
+  ![Screenshot showing resource group being created](./02.png)
+
+* Create a unique name for the FQDNs
+
+  The fully qualified domain names must be globally unique.
+
+  ```shell
+  name=arcmi$(az account show --query id --output tsv | cut -f1 -d-)
+  echo "Name: $name"
+  ```
+
+  The command uses the first eight characters from your subscription ID and suffixes to the string `arcmi` to create a name for the FQDNs.
+
+  ![Screenshot showing an example unique name for the FQDNs](./03.png)
```

Comment:
Mask subscription id

Reply:
I pixellated the subscription GUID after the first hyphen, as the first eight characters are used in the command to create the unique name for the FQDNs.